<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_code_fournisseur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Code du fournisseur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code_fournisseur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_code_fournisseur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrer la lettre F suivie de cinq caractères alphanumériques</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_compagnie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom de la compagnie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_compagnie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelErreur_nom_compagnie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alphanumérique de lomgueur inférieur à 30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_patient">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Patient</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_patient">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ajouter_listview">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajouter le patient à la listview</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_list_view">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>La listview qui contient la liste des patients qui achètent les médicaments de ce fournisseur</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>431</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_serialiser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sérialiser fournisseur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Désérialiser fournisseur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
